cgi-bin/LJ/Console/Command/EmailAlias.pm

```diff
@@ -45,7 +45,10 @@ sub execute {
 
     # canonicalize
     $alias =~ s/\@.*//;
+    my $alias_with_dashes = $alias;
```

Comment:
Because the alias given in the command may not correspond to a user object at all.